dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -173,7 +175,7 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 	}
 	for _, path := range pathsToCopy {
 		mkdir(filepath.Join(dstPath, filepath.Dir(path)))
-		run(fmt.Sprintf("adding %v", path), "mv", path, filepath.Join(dstPath, path))
+		run(fmt.Sprintf("adding %v", path), "cp", path, filepath.Join(dstPath, path))
```

Comment:
why `cp` instead of `mv?

Reply:
it resolves symlinks, while mv preserves them. We don't want the templates to be symlinks because docker can't handle symlinks that point outside the build context